### PR TITLE
WIP: Id issue7

### PR DESCRIPTION
**Pull request by pradal**
Close Issue #7 

@bilalderouich

### CODE REVIEW

**Comment by christian34** [comment]
I just push my commits for the finalisation of the externalisation of id management. @pradal @Emblanc : For me its is now ok for merging / review.
Be careful: I move the walter_command code in a new module: as a result, python setup.py develop should be run again before testing.

**Comment by pradal** [comment]
@christian34 
I will rename water_command to main.py or command.py
Because the module is walter, there is no need to prefix the module by walter_

File: src/walter/walter_command.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+    if args.i:
+        if not sim_scheme:
+            prj.run(dry_run=args.dry_run)
+        else:
+            param_list = prj.csv_parameters(sim_scheme)
+            if len(param_list) == 1:
+                prj.run(dry_run=args.dry_run, **(param_list[0]))
+            else:
+                print 'Multiple processes'
```
Comment (pradal) [inline src/walter/walter_command.py]: Add parenthesis around print stment.

File: src/walter/walter_command.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+    prj = project.Project(p)
+
+    # TODO: add a flag in the project to know if the project has been generated, modified or not.
+    if prj.dirname.exists():
+        print('Use Project %s located at %s' % (prj.name, prj.dirname))
+    else:
+        print('Project %s has been generated at %s' % (prj.name, prj.dirname))
+
+    if args.i:
```
Comment (pradal) [inline src/walter/walter_command.py]: I do not understand these two lines: if args.i then sim_scheme == args.i

File: src/walter/walter_command.py [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    parser = walter_parser()
+    args = parser.parse_args()
+
+    p = '.'
+    if args.p:
+        p = args.p
+    else:
+        if not project.check_cwd():
+            answer = raw_input("Current directory doesn't look like a walter project dir, Continue ? [Y/N]")
+            if not answer == 'Y':
```
[the remark this answers is not in the document]
Comment (christian34) [reply src/walter/walter_command.py]: done

File: src/walter/walter_command.py (hunk 1 above)
Comment (christian34) [reply src/walter/walter_command.py]: done

File: src/walter/walter_command.py (hunk 2 above)
Comment (christian34) [reply src/walter/walter_command.py]: that was a mistake de to missunderstanding of parser.
 I put the correct logic in walter_parser, update command doc and add a test in test_command_line.py: 
-p only : do not run anything
-i : run walter without args (lpy defaults)
-i my_sim.csv;: run walter with parameters in my_sim.csv

**Comment by pradal** [comment]
Thanks Emmanuelle and Christian.
Great work.

**Pull request merged**